api/HTMLInputElement.json

```diff
@@ -289,9 +289,9 @@
           }
         }
       },
-      "formEncType": {
+      "formEnctype": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLInputElement/formEncType",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLInputElement/formEnctype",
```

Comment:
Both this and the old URL is 404. @ddbeck should we include the URLs in this case?

Reply:
Dealing with 404s is a whole… thing. #5297 is open to remove them, but there's some process questions around that (how do we pick up those pages once they're created?) that are unanswered. For now, I suggest keeping the 404s.